src/plugins/reviewDB/index.tsx

```diff
@@ -47,14 +47,29 @@ const guildPopoutPatch: NavContextMenuPatchCallback = (children, props: { guild:
     );
 };
 
+const userContextPatch: NavContextMenuPatchCallback = (children, props: { user?: User, onClose(): void; }) => {
```

Comment:
```suggestion
const userContextPatch: NavContextMenuPatchCallback = (children, { user }: { user?: User, onClose(): void; }) => {
```

then you dont need the non null assertions

Reply:
i don't understand how destructuring will fix that? if you look at the commit history you will see why the check was even implemented in the first place

Reply:
![image](https://github.com/Vendicated/Vencord/assets/37475903/5ce5220e-422e-4178-b03e-2a450b49d8d3)

Reply:
ImageZoom also has a similar issue

Reply:
i resolved it myself